lib/buildingsync/makers/workflow_maker.rb

```diff
@@ -364,6 +364,13 @@ def write_osws(facility, dir)
           scenario_type_element.add_element(package_of_measures_element)
           scenario_element.add_element(scenario_type_element)
 
+          # adding XML elements for the new way to define a baseline scenario
```

Comment:
This doesn't look like it would work as expected.  It looks like its adding both a:
- package of measures  scenario type
- current building modeled scenario type

Fix.

Reply:
good catch, fixed here https://github.com/BuildingSync/BuildingSync-gem/commit/bb00d9f157ed812d41bb2ce317bf16253787b861